백다연/MvvmStudy/app/src/main/java/com/test/mvvmstudy/view/SearchFragment.kt

```diff
@@ -0,0 +1,88 @@
+package com.test.mvvmstudy.view
+
+import android.os.Bundle
+import android.view.*
+import androidx.appcompat.widget.SearchView
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.viewModels
+import androidx.navigation.Navigation
+import com.test.mvvmstudy.R
+import com.test.mvvmstudy.adapter.SearchResultAdapter
+import com.test.mvvmstudy.databinding.FragmentSearchBinding
+import com.test.mvvmstudy.data.ResultDetail
+import com.test.mvvmstudy.viewmodel.SearchViewModel
+
+class SearchFragment : Fragment() {
+    private lateinit var binding: FragmentSearchBinding
+    private val adapter: SearchResultAdapter by lazy {
+        SearchResultAdapter()
+    }
+    private val searchViewModel: SearchViewModel by viewModels()
+
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        binding = FragmentSearchBinding.inflate(inflater, container, false)
+
+        return binding.root
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        setHasOptionsMenu(true)
+        setRecyclerviewAdapter()
+        adapterClickListener()
+        initObserver()
+    }
+
+    private fun setRecyclerviewAdapter() {
+        binding.searchRecyclerview.adapter = adapter
+    }
+
+    private fun adapterClickListener() {
```

Comment:
```suggestion
    private fun setAdapterClickListener() {
```

함수가 동사로 시작하는 것이 좋아보입니다!! 네이밍이라 참고만 해주세요 :)